utilities/migration_utils.py

```
@@ -231,6 +234,24 @@ def prepare_migration_for_tests(
         for idx in range(len(virtual_machines_list)):
             virtual_machines_list[idx].update({"namespace": source_vms_namespace})
 
+    # Prepare target_node_selector
+    target_node_selector = None
+    if plan.get("target_node_selector") and labeled_worker_node is not None:
+        if "label_key" not in labeled_worker_node or "label_value" not in labeled_worker_node:
+            raise ValueError("labeled_worker_node missing required keys 'label_key' or 'label_value'")
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)